docs/setup/03 - Project Setup.md

```diff
@@ -186,7 +196,11 @@ See the following json-snippet as an example:
         }
 }
 ```
-You see that there are multiple task levels. If you only want to go for classification, keep task level 1 in the config. In case you do not want to integrate Multi Label Classification, Named Entity Recognition, Question/Answering, Opinion Mining..., simply remove it from your JSON.
+You see that there are multiple task levels. To give you a bit of help here, use following examples:
```

Comment:
Maybe I just don't get it, but why are you talking about "levels" here? I only see task 1, task 2, ... 
Using the terms level, I would expect something like that:
```json
"tasks": {
   "classification": [
      {
            "label": "subcat",
            "type": "classification",
            "model_type": "roberta",
            "max_seq_len": 256,
            "embeds_dropout":0.3,
            "learning_rate":3e-5,
            "prepare": true
        },
        "2": {
            "type": "multi_classification",
            "model_type": "roberta",
            "max_seq_len": 256,
            "embeds_dropout":0.3,
            "learning_rate":2e-5,
            "prepare": true
        }
   ],
   "questionAnswering": []
...
```